build/Dockerfile

```diff
@@ -79,6 +79,7 @@ RUN apt-get update && \
         vim \
         xxd \
         cmake \
+        python3-yaml \
```

Comment:
maybe add also comment above regarding why it's needed (eg. ` python3-yaml: phoenix-rtos-build tools`)

Reply:
Done. Also added python3 package just in case

Reply:
in the comment above please - where all of the other deps are described:
![image](https://github.com/phoenix-rtos/phoenix-rtos-docker/assets/3939983/2d792b4d-381d-4782-bad4-9bca06186e4a)